logs.go

```diff
@@ -91,6 +92,23 @@ func getLoggableResources(getPodContainers func(string, string) ([]string, error
 	return loggableResources, nil
 }
 
+// GetProjectPodsBySubstr returns a list of pod names that contain the provided string, in the named project.
+func getProjectPodNamesBySubstr(project, substr string) ([]string, error) {
+	ret := []string{}
+	pods, err := GetResourceNames(project, "pod")
+	if err != nil {
+		return nil, err
+	}
+
+	for _, pod := range pods {
+		if strings.Contains(pod, substr) {
+			ret = append(ret, pod)
+		}
+	}
+
+	return ret, nil
```

Comment:
@vchepeli There is no `Error.NIL` in go, returning `nil` for errors when no error has occurred is a pretty common usage, see here for more info: https://golang.org/doc/effective_go.html#multiple-returns